Source/WebCore/Modules/encryptedmedia/InitDataRegistry.cpp

```diff
@@ -41,7 +41,11 @@ namespace WebCore {
 
 namespace {
     const uint32_t kCencMaxBoxSize = 64 * KB;
-    const uint8_t playreadyXMLMagic[10] = { 0x8c, 0x03, 0x00, 0x00, 0x01, 0x00, 0x01, 0x00, 0x82, 0x03 };
+    constexpr uint8_t playreadyXMLMagicLength = 20;
+    constexpr uint8_t playreadyXMLMagicOffset = 10;
+    // L("<WRMHEADER")
+    const uint8_t playreadyXMLMagic[playreadyXMLMagicLength] = { 0x3c, 0x00, 0x57, 0x00, 0x52, 0x00, 0x4d, 0x00, 0x48, 0x00, 0x45, 0x00, 0x41, 0x00, 0x44, 0x00, 0x45, 0x00, 0x52, 0x00 };
```

Comment:
You can just declare:

```
const uint8_t playreadyXMLMagic[] = { 0x3c, 0x00, 0x57, 0x00, 0x52, 0x00, 0x4d, 0x00, 0x48, 0x00, 0x45, 0x00, 0x41, 0x00, 0x44, 0x00, 0x45, 0x00, 0x52, 0x00 };
```

and use `sizeof(playreadyXMLMagic)` when needed. We wouldn't need `playreadyXMLMagicLength`